AK/Base64.cpp

```diff
@@ -5,13 +5,7 @@
  */
 
 #include <AK/Array.h>
-#include <AK/Assertions.h>
 #include <AK/Base64.h>
-#include <AK/CharacterTypes.h>
-#include <AK/Error.h>
-#include <AK/StringBuilder.h>
-#include <AK/Types.h>
-#include <AK/Vector.h>
```

Comment:
That's too aggressive. There's uses of at least `StringBuilder` and `Vector` in the file below, these are not "unused".

Reply:
Right, well spotted. CLion insists that its unused and the build works without it. Perhaps they are transitively included? Doesn't bode well for my approach here then :(

Reply:
As confirmed by @Hendiadyoin1, they are transitive includes, which I agree is a bad idea to remove. Not sure this approach can be fixed to not remove them, unless I can make CLion smarter. Oh well, perhaps it could be useful for someone cleverer in the future.

Reply:
Actually, CLion isn't supposed to remove them. I chose the strategy "Detect not directly used (default)" which from the documentation  should "follows the Include What You Use principle (the principle that if you use a symbol or type from a header, you should include that header) and detects the #includes directives with the declarations not used in the current file directly.". I can try picking the most conservative option:  "Detect completely unused: the most conservative strategy, which detects only the #include directives with the declarations never used in the translation unit."

Reply:
> I can try picking the most conservative option

Sounds good, let me know when you've changed it and I'll have another look :)

Reply:
It appears the conservative option still removes the dependencies you commented on here :(